runner/runners/queue.go

```diff
@@ -327,6 +327,7 @@ func (c *QueueController) loop() {
 	var watchCh chan runner.RunStatus
 	var updateDoneCh chan interface{}
 	updateRequested := false
+	var idleLatency = c.inv.stat.Latency(stats.WorkerIdleLatency_ms).Time()
```

Comment:
[nit] assign "c.inv.stat.Latency(stats.WorkerIdleLatency_ms)" to a new variable so it isn't accessed 3 separate times.

Reply:
Yup, agree!